### PR TITLE
Remove WSC

### PR DESCRIPTION
**Pull request by jacekbogdanski**
WSC approached its end-of-life (EOL) on December 31, 2021 and should be removed from documentation.

Please, wait for merging this PR until further notice (although feel free to review it).

### CODE REVIEW

**Comment by Comandeer** [comment]
Rebased onto latest `master`.  Also [rephrased one sentence](https://github.com/ckeditor/ckeditor4-docs/pull/399/commits/a5467c2b9ba94d932c07fcaaff64c15f6c79e174).

File: docs/guide/dev/patching/README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ There are two repositories where CKEditor 4 source files are kept: [`ckeditor4`]
 The `ckeditor4-presets` repository is used by the CKEditor 4 team to build the Basic/Standard/Full distributions. It uses `ckeditor4` as a dependency and scripts included there further automate the build process:
 
  - It has information about which plugins should be included in which preset.
- - It loads spell checker plugins (`scayt` and `wsc`) from separate repositories where they are developed, if they are to be included in a release.
+ - It loads spell checker `scayt` plugin from separate repository where it's developed, if it has to be included in a release.
```
Comment (Comandeer) [inline docs/guide/dev/patching/README.md:61]:
```suggestion
 - It loads spell checker `scayt` plugin from separate repository where it is developed, if it is to be included in a release.
```

File: docs/guide/dev/patching/README.md [hunk 2]
```diff
@@ -186,9 +186,9 @@ It is possible that `ckeditor4/plugins` does not contain all plugins that your b
 
 If you used a custom build of CKEditor 4 with your own custom plugins, copy them to the `ckeditor4/plugins` folder.
 
-#### Spell Checker Plugins
+#### Spell Checker Plugin
 
-If the build that you used had spell checker plugins (`scayt` or `wsc`), then you need to copy them as well. Spell checker plugins must be downloaded with the proper revision. The revision of `scayt` and `wsc` plugin that was used with release versions of CKEditor 4 can be checked in the `ckeditor4-presets` repository
+If the build that you used had spell checker plugin (`scayt`), then you need to copy them as well. Spell checker plugin must be downloaded with the proper revision. The revision of `scayt` plugin that was used with release versions of CKEditor 4 can be checked in the `ckeditor4-presets` repository
```
Comment (Comandeer) [inline docs/guide/dev/patching/README.md:191]:
```suggestion
If the build that you used had the spell checker plugin (`scayt`), then you need to copy it as well. Spell checker plugin must be downloaded with the proper revision. The revision of `scayt` plugin that was used with release versions of CKEditor 4 can be checked in the `ckeditor4-presets` repository.
```

File: docs/guide/dev/patching/README.md [hunk 3]
```diff
@@ -206,11 +206,11 @@ If the build that you used had spell checker plugins (`scayt` or `wsc`), then yo
 
     <img src="%BASE_PATH%/assets/img/patching_05.png" width="391" height="181" alt="Plugin revision in GitHub">
 
-6. Click both links to exact revisions of the `scayt` and `wsc` plugins. You will be redirected to their project pages, where the "Download ZIP" option on the right side will return the correct version of each plugin.
+6. Click the link to exact revision of the `scayt` plugin. You will be redirected to its project page, where the "Download ZIP" option on the right side will return the correct version of each plugin.
```
Comment (Comandeer) [inline docs/guide/dev/patching/README.md:209]:
```suggestion
6. Click the link to exact revision of the `scayt` plugin. You will be redirected to its project page, where the "Download ZIP" option on the right side will return the correct version of the plugin.
```

File: docs/guide/dev/patching/README.md [hunk 4]
```diff
@@ -206,11 +206,11 @@ If the build that you used had spell checker plugins (`scayt` or `wsc`), then yo
 
     <img src="%BASE_PATH%/assets/img/patching_05.png" width="391" height="181" alt="Plugin revision in GitHub">
```
Comment (Comandeer) [inline docs/guide/dev/patching/README.md:207]: I'm not sure if it wouldn't be better to update the screen to remove the WSC plugin submodule from it :thinking: 

Also, this method does no longer works as we now load SCAYT as npm dependency.

File: docs/sdk/examples/spellchecker.html [hunk 5]
```diff
@@ -37,7 +37,7 @@ <h1>Proofreading, Spelling and Grammar Checking <a class="documentation" href="h
 
 			<p>
 				The Standard and Full CKEditor 4 packages include the integrated spell checking functionality.
-				It is an out-of-the-box solution provided by third-party plugins, <a href="https://ckeditor.com/cke4/addon/wsc">WebSpellChecker</a> and <a href="https://ckeditor.com/cke4/addon/scayt">SpellCheckAsYouType (SCAYT)</a>, that use the <a href="https://webspellchecker.com">WebSpellChecker</a> web service.
+				It is an out-of-the-box solution provided by third-party plugin, <a href="https://ckeditor.com/cke4/addon/scayt">SpellCheckAsYouType (SCAYT)</a>, that use the <a href="https://webspellchecker.com">WebSpellChecker</a> web service.
```
Comment (Comandeer) [inline docs/sdk/examples/spellchecker.html:40]:
```suggestion
				It is an out-of-the-box solution provided by the third-party plugin, <a href="https://ckeditor.com/cke4/addon/scayt">SpellCheckAsYouType (SCAYT)</a>, that uses the <a href="https://webspellchecker.com">WebSpellChecker</a> web service.
```

**Review by Comandeer — CHANGES_REQUESTED** [review]

File: docs/guide/dev/patching/README.md (hunk 4 above)
Comment (jacekbogdanski) [reply docs/guide/dev/patching/README.md:207]: Yeah, you are right. I didn't notice how outdated this guide is. Anyway, I would be for extracting it as a separate issue so we will be able to test these changes during the testing phase, WDYT?

Regarding the screenshots themselves, fortunately, there is also `scayt` on them so I think we can just move that part also to the separate issue addressing the whole guide.

Comment (jacekbogdanski) [reply docs/guide/dev/patching/README.md:207]: Extracted https://github.com/ckeditor/ckeditor4/issues/5117

**Review by Comandeer — APPROVED** [review]
LGTM!

**Pull request merged**